test/orderbookReserve.js

```diff
@@ -4625,7 +4625,7 @@ contract('OrderbookReserve_feeBurner_network', async (accounts) => {
 
         ethKncRate = initialEthKncRate * 2;
         let ethToKncRatePrecision = precisionUnits.mul(ethKncRate);
-        let kncToEthRatePrecision = precisionUnits.div(ethKncRate);
+        let kncToEthRatePrecision = precisionUnits.div(ethKncRate * 1.01);
```

Comment:
how is this change related?

Reply:
Related to the feeBurner PR. Due to numerical precision, it's failing the arbitrage check in the fee burner.